update README.md (#58)
delta-io/connectors: README.md

#### SBT
```

> libraryDependencies += "io.delta" %% "delta-standalone" % "0.2.0-SNAPSHOT"

Reviewer: We prefer to show a release version rather than a snapshot version in README so that people can try it.

Author: it doesn't work without the "-SNAPSHOT" when I run it locally. Is that how it is supposed to work?

Reviewer: We usually put the last release version here. But because there is no release version available right now, it will fail. Once we have the first release, it should work.

Author: makes sense, thanks

SET hive.tez.input.format=io.delta.hive.HiveInputFormat;
```

#### Add Hive uber JAR

The second step is to upload the above uber JAR to the machine that runs Hive. Next, make the JAR accessible to Hive. There are several ways to do this, listed below. To verify that the JAR was properly added, run `LIST JARS;` in the Hive CLI.

- in the Hive CLI, run `ADD JAR <path-to-jar>;`

> - move the uber JAR to the folder already pointed to by the `HIVE_AUX_JARS_PATH` environmental variable. (Its location is `/usr/lib/hive-hcatalog/share/hcatalog` in an EMR cluster)

Reviewer: ```suggestion
- add the uber JAR to a folder already pointed to by the `HIVE_AUX_JARS_PATH` environmental variable.
```

`/usr/lib/hive-hcatalog/share/hcatalog` is not a great place to put this jar. It's just a hack I did. It's better to not document it.

> - modify the same `/etc/hive/conf/hive-site.xml` file as above, and add the following. (Note that this has to be done before you start the Hive CLI)

Reviewer: ```suggestion
- modify the same `hive-site.xml` file as above, and add the following. (Note that this has to be done before you start the Hive CLI)
```

If the schema in the underlying Delta metadata is not consistent with the schema specified by `CREATE TABLE` statement, the connector will report an error when loading the table and ask you to fix the schema. You must drop the table and recreate it using the new schema. Hive 3.x exposes a new API to allow a data source to hook ALTER TABLE. You will be able to use ALTER TABLE to update a table schema when the connector supports Hive 3.x.

#### Hive has three execution engines, MapReduce, Tez and Spark. Which one does this connector support?

> The connector supports Tez (default) and MapReduce. It doesn't support Spark execution engine in Hive.

Reviewer: Let's revert chanage. `Tez` is default on EMR because EMR changes it. The real default value is `MapReduce` in official Apache Hive 2.x.
